config/v1/types_authentication.go

```diff
@@ -192,6 +192,14 @@ type OIDCProvider struct {
 	// +required
 	Issuer TokenIssuer `json:"issuer"`
 
+	// OIDCClients contains configuration for the platform's clients that
+	// need to request tokens from the issuer
+	//
+	// +listType=map
+	// +listMapKey=component.name
```

Comment:
@deads2k - would you know if this actually works this way?

Reply:
dont' know.  You'll need to create it on real cluster and try using `oc apply`

Reply:
Unfortunately, nesting does not seem to work. I inlined it to `componentName` and `componentNamespace`